app/marketing/mails.py

```diff
@@ -866,7 +844,7 @@ def notify_deadbeat_quest(quest):
         setup_lang(to_email)
         subject = _("Dead Quest Alert")
         body = f"This quest is dead ({quest.title}): https://gitcoin.co/{quest.admin_url} "
-        if not should_suppress_notification_email(to_email, 'faucet'):
```

Comment:
Should we just remove these entirely rather than changing to the quest or kudos emails?

Reply:
hmm but this would mean that the quest email never gets fired ! is that what we want ?
It was mapped to the wrong setting from what i can tell